pkg/virt-controller/watch/migration/migration.go

```diff
@@ -137,6 +138,7 @@ func NewController(templateService services.TemplateService,
 	storageProfileInformer cache.SharedIndexInformer,
 	migrationPolicyInformer cache.SharedIndexInformer,
 	resourceQuotaInformer cache.SharedIndexInformer,
+	kubevirtInformer cache.SharedIndexInformer,
```

Comment:
I see there is no check that ensures this informer has synced.
Actually, I see there are also other not checked:
- nodeInformer 
- pvcInformer
- storageClassInformer
- storageProfileInformer
- migrationPolicyInformer
- kubevirtInformer

These should be added in the hasSynced function, and the sanityExecute in the related unit test.
```golang
	c.hasSynced = func() bool {
		return vmiInformer.HasSynced() && podInformer.HasSynced() && migrationInformer.HasSynced() && resourceQuotaInformer.HasSynced()
	}
```